scripts/dockcross-manylinux-build-module-wheels.sh

```diff
@@ -18,7 +18,7 @@
 #
 
 MANYLINUX_VERSION=_2_28
-IMAGE_TAG=20221128-2024e4b
+IMAGE_TAG=20221108-102ebcc
```

Comment:
I am not sure that this is the the right solution.

Reply:
I have confirmed that 3.9 packages build on my local system with this change, though it seems 3.7 is still failing in ITKSplitComponents CI. Will continue to investigate.

Reply:
@dzenanz I have further documented the issue in https://github.com/dockcross/dockcross/issues/746. My guess is that there will be a small fix requiring a new manylinux image to be tagged. For now reverting to an earlier image is required to allow Linux Python builds to succeed. I have kept the revert as a separate commit that can be easily reverted in the case that a solution is introduced that uses the 20221128-2024e4b tag.